src/contribute/release-guide.md

```diff
@@ -275,24 +277,39 @@ By default, the Javadoc will be generated in `target/site/apidocs/`. Let `${JAVA
 
 Please carefully review the generated Javadoc. Check for completeness and presence of all relevant packages and `package-info.java`; consider adding less relevant packages to the `excludePackageNames` configuration. The index page is generated at `${JAVADOC_ROOT}/index.html`.
 
+#### Generate Pydoc
+
+Create the Python SDK documentation using sphinx by running a helper script.
+```
+cd sdks/python && ./generate_pydocs.sh
+```
+By default the Pydoc is generated in `sdks/python/docs/_build`. Let `${PYDOC_ROOT}` be the absolute path to `_build`.
+
 ### Propose a pull request for website updates
 
 The final step of building the candidate is to propose a website pull request.
 
 Start by updating `release_latest` version flag in the top-level `_config.yml`, and list the new release in the [Apache Beam Downloads]({{ site.baseurl }}/get-started/downloads/), linking to the source code download and the Release Notes in JIRA.
 
+#### Create Javadoc
 Add the new Javadoc to [SDK API Reference page]({{ site.baseurl }}/documentation/sdks/javadoc/) page, as follows:
 
 * Copy the generated Javadoc into the website repository: `cp -r ${JAVADOC_ROOT} documentation/sdks/javadoc/${VERSION}`.
 * Update the Javadoc link on this page to point to the new version (in `src/documentation/sdks/javadoc/current.md`).
 
+#### Create Pydoc
+Add the new Pydoc to [SDK API Reference page]({{ site.baseurl }}/documentation/sdks/pydoc/) page, as follows:
+
+* Copy the generated Pydoc into the website repository: `cp -r ${PYDOC_ROOT} documentation/sdks/pydoc/${VERSION}`.
+* Update the Pydoc link on this page to point to the new version (in `src/documentation/sdks/pydoc/current.md`).
+
 Finally, propose a pull request with these changes. (Don’t merge before finalizing the release.)
 
 ### Checklist to proceed to the next step
 
 1. Maven artifacts deployed to the staging repository of [repository.apache.org](https://repository.apache.org/content/repositories/)
 1. Source distribution deployed to the dev repository of [dist.apache.org](https://dist.apache.org/repos/dist/dev/beam/)
-1. Website pull request proposed to list the [release]({{ site.baseurl }}/use/releases/) and publish the [API reference manual]({{ site.baseurl }}/learn/sdks/javadoc/)
+1. Website pull request proposed to list the [release]({{ site.baseurl }}/use/releases/), publish the [Java API reference manual]({{ site.baseurl }}/learn/sdks/javadoc/), and publish the [Python API reference manual]({{ site.baseurl }}/learn/sdks/pydoc/).
```

Comment:
Ack. Do the current canonical URL for anything new, and update any old stuff that you encounter by accident. Otherwise, don't worry about it. (If you do a non-canonical one, page load time (unfortunately) ~doubles.)